src/event.js

```diff
@@ -573,7 +574,7 @@ jQuery.Event.prototype = {
 
 		this.isPropagationStopped = returnTrue;
 
-		if ( e ) {
+		if ( e && this.executeOriginalPropagation ) {
```

Comment:
> But rereading the simulate comments,

If we would go to that route we would need the same thing for the `preventDefault`.

I think it was and it is reasonable, back then i checked with the spec of all donor events in case they had default actions, cancelable, etc; they all supposed to be noop - still holds true now too.

So it just waist of the space and judging by conversation of the parallel thread this is indeed mozilla's bug.

So these lines could be only "just in case" lines, without tests or use case, when/if mozilla people comes up with the fix